group_vars/all/vars.yml

```diff
@@ -1,5 +1,4 @@
 ---
-ruby_version: "{{ ruby_version_default }}"
```

Comment:
I wondered about this myself but wasn't brave enough to remove the hard linked 2.3.1